app/models/address.rb

```diff
@@ -2,5 +2,5 @@ class Address < ActiveRecord::Base
   belongs_to :state
   belongs_to :user
   has_many :orders, inverse_of: :address
-  validates_presence_of :name, :line1, :city, :state, :zip, :user
+  validates_presence_of :name, :line1, :city, :state, :zip, :user_id
```

Comment:
@fiestacasey I changed it because the update function wouldnt work on addresses and it did work on credit cards.  This was the only difference I could really see, and when I changed it to user_id rather than just :user it worked.